### PR TITLE
fly: add network env var

### PR DESCRIPTION
**Pull request by panoel**

### CODE REVIEW

File: fly/cmd/fly/main.go [hunk 1]
```diff
@@ -140,8 +142,16 @@ func verifyEnvVar(key string) string {
...
-	p2pBootstrap = p2p.MainnetBootstrapPeers
+	if network == "mainnet" {
+		p2pNetworkID = p2p.MainnetNetworkId
+		p2pBootstrap = p2p.MainnetBootstrapPeers
+	} else if network == "testnet" {
+		p2pNetworkID = p2p.TestnetNetworkId
+		p2pBootstrap = p2p.TestnetBootstrapPeers
+	} else {
+		p2pNetworkID = p2p.DevnetNetworkId
+		p2pBootstrap = ""
```
Comment (bruce-riley) [inline fly/cmd/fly/main.go:153]: Should you fatal error here, since it's not going to work without the bootstrap?

File: fly/cmd/fly/main.go [hunk 2]
```diff
@@ -140,8 +142,16 @@ func verifyEnvVar(key string) string {
 
 func main() {
 	loadEnvVars()
-	p2pNetworkID = p2p.MainnetNetworkId
-	p2pBootstrap = p2p.MainnetBootstrapPeers
+	if network == "mainnet" {
```
Comment (evan-gray) [inline fly/cmd/fly/main.go:145]: Can this be consistent with other services and do something like

```suggestion
	env, err := common.ParseEnvironment(network)
	if err != nil || (env != common.TestNet && env != common.MainNet) {
		logger.Fatal("Invalid value for NETWORK, should be testnet or mainnet", zap.String("val", network))
	}
	p2pNetworkID = p2p.GetNetworkId(env)
	p2pBootstrap, err = p2p.GetBootstrapPeers(env)
	if err != nil {
		logger.Fatal("failed to determine the bootstrap peers from the environment", zap.String("env", string(env)), zap.Error(err))
	}
```

(this example moves it after the logger setup)

Comment (panoel) [reply fly/cmd/fly/main.go:145]: Changed

File: fly/cmd/fly/main.go (hunk 1 above)
Comment (panoel) [reply fly/cmd/fly/main.go:153]: Changed

**Review by evan-gray — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**